### PR TITLE
Gm no reformat

### PR DESCRIPTION
**Pull request by tomkoren21**

### CODE REVIEW

File: onnx2kerastl/reshape_layers.py [hunk 1]
```diff
@@ -65,6 +66,45 @@ def convert_shape(node, params, layers, lambda_func, node_name, keras_name):
...
+            # [0,1,2,3,0,1,2,3]
+            logger.debug('onnx2keras.gather - Shortening sequence - columns')
+            indices = indices % (max_inp + 1)
+            params = range_inp
+        else:
+            repetition_len = np.argmin(params == params[0])
+            if repetition_len > 0 and len(params) % repetition_len == 0:
+                reshaped_rows = tf.reshape(params, [-1, repetition_len])
+                first_row = reshaped_rows[:, 0]
+                if tf.reduce_sum(tf.abs(reshaped_rows - first_row[:, None])) == 0:
```
Comment (RanHomri) [inline onnx2kerastl/reshape_layers.py:102]: Maybe add a line comment for each if/else to explain in plain words what is it checking

File: onnx2kerastl/operation_layers.py [hunk 2]
```diff
@@ -773,12 +773,16 @@ def convert_nms(node, params, layers, lambda_func, node_name, keras_name):
             score_threshold = score_threshold[0]
     num_classes = scores.shape[1]
     all_results = []
+    try:
+        iou_threshold = iou_threshold[0]
+    except IndexError: #iou threshold is already a scalar
+        pass
```
Comment (RanHomri) [inline onnx2kerastl/operation_layers.py:779]: We are using `ensure_float` on `score_threshold` for the same reason. why not using here as well?

**Review by RanHomri — APPROVED** [review]
I left two comments but they are not a must

File: onnx2kerastl/operation_layers.py (hunk 2 above)
Comment (tomkoren21) [reply onnx2kerastl/operation_layers.py:779]: What if the IOU_THRESHOLD is dynamically computed? i.e. a KerasTensor and not an Eager tensor? Currently, ensure_float will not deal properly with that IMO..

Suggest we hold this off until we make some improvement to ensure_float and test it and then maybe standartize it across Onnx2kerras

File: onnx2kerastl/reshape_layers.py (hunk 1 above)
Comment (tomkoren21) [reply onnx2kerastl/reshape_layers.py:102]: Done

**Pull request merged**